unitTests/ui/MissingTermManagerTest.ts

```diff
@@ -0,0 +1,65 @@
+import * as Mock from '../MockEnvironment';
+import { MissingTermManager } from '../../src/ui/MissingTerm/MissingTermManager';
+import { Simulate } from '../Simulate';
+import { $$, l } from '../Test';
+
+export function MissingTermsManagerTest() {
+  describe('MissingTermManager', () => {
+    let env: Mock.IMockEnvironment;
+    let arrayGet: Array<string>;
```

Comment:
`string[]` would do.

Reply:
The `arrayGet` name is not super descriptive...

How about just `missingTerms` ? 

Tests would then read like this: 
```
missingTerms.push('is')
simulateQuery()
expect(query.advanced[.....]).toContain('is')
```